src/features/Dashboard/AssignedClasses/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+
+import { Col } from '@edx/paragon';
+import { Button } from 'react-paragon-topaz';
+
+import ClassCard from 'features/Dashboard/AssignedClasses/ClassCard';
+
+import 'features/Dashboard/AssignedClasses/index.scss';
+
+const AssignedClasses = () => {
+  const classes = useSelector((state) => state.common.allClasses.data);
+  const [classCards, setClassCards] = useState([]);
+  const numberOfClasses = 3;
```

Comment:
`numberOfClasses` the name is not very clear, what about something like: `visibleClassCount` or `displayClassLimit`.